### ISSUE TITLE
Support matrix parameters

### ISSUE THREAD

**Issue by t1** [raml-org/raml-spec#187]
JAX-RS allows to specify `@MatrixParam`s. Matrix parameters are mentioned as a common practice in the last paragraph of section 3.3 [rfc-3986](http://tools.ietf.org/html/rfc3986#section-3.3), although not formally specified as part of that rfc or named as matrix parameters. The [original description](http://www.w3.org/DesignIssues/MatrixURIs.html) of the concept is from Tim Berners-Lee from 1996.

Adding support for matrix parameters to the raml-spec would be an important step to be able to use them properly.

As the Swagger team seems to [be unwilling](https://github.com/swagger-api/swagger-spec/issues/69) to add support for matrix parameters to their spec, this is one of the use-cases to switch to raml ;-)

**Comment by kernixski** [comment]
I agree: adding support for matrix parameters could give an advantage to RAML over Swagger.  For now, only WADL support matrix parameters... :o/

**Comment by kernixski** [comment]
Does anybody know when RAML specs will support matrix parameters?

Here is an example of usage:

GET http://my.domain.com/api/v1/orders;min_amount=1000/persons (will list the persons that have made at least one order with a minimal amount of 1.000$)
